frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/clearingStatus.jspf

```diff
@@ -71,8 +73,9 @@
             columns: [
                 {title: "Name"},
                 {title: "Project Origin"},
-                {title: "Release Clearing State"},
-                {title: "Release Mainline State"},
+                {title: "Clearing State"},
+                {title: "Mainline State"},
+                {title: "Project Mainline State"},
```

Comment:
The change is:
`Release Clearing State` -> `Clearing State`
`Release Mainline State` -> `Mainline State`
new column `Project Mainline State`